code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -166,6 +166,10 @@
 	max_temperature = T20C
 	min_temperature = 170
 	circuit = /obj/item/circuitboard/machine/thermomachine/freezer
+	
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer/on
+	on = TRUE
+	target_temperature = 73.15
```

Comment:
line 167 `min_temperature = 170`?

Reply:
Freezers with stock parts cool up to 73.15K, even though min_temperature is 170K. No clue why, maybe RefreshParts() causes it?

Reply:
Maps used to have roundstart target_temp defined as 80K, doesn't make much sense imo when the default mininum temp is 73.15K.